spec/unit/facts/network_nexthop_ip_spec.rb

```diff
@@ -8,6 +8,7 @@
   end
   context 'on a Linux host' do
     before do
+      Facter.fact(:virtual).stubs(:value).returns('physical')
```

Comment:
this appeared necessary in my setup, i don't know if it isn't actually a sign of… some breakage… to come.